clients/java/dkv-client/src/main/java/org/dkv/client/SimpleDKVClient.java

```diff
@@ -138,6 +138,46 @@ public void put(byte[] key, byte[] value, long expiryTS) {
         put(copyFrom(key), copyFrom(value), expiryTS);
     }
 
+    @Override
+    public void put(KV.Strings... items) {
+        Api.MultiPutRequest.Builder multiPutReqBuilder = Api.MultiPutRequest.newBuilder();
+        for (KV.Strings kv : items){
+            Api.PutRequest.Builder putReqBuilder = Api.PutRequest.newBuilder();
+            Api.PutRequest putReq = putReqBuilder
+                    .setKey(copyFromUtf8(kv.getKey()))
+                    .setValue(copyFromUtf8(kv.getValue()))
+                    .setExpireTS(kv.getExpiryTS())
+                    .build();
+            multiPutReqBuilder.addPutRequest(putReq);
+        }
+        Api.MultiPutRequest multiPutReq = multiPutReqBuilder.build();
+        Api.PutResponse response = blockingStub.multiPut(multiPutReq);
+        Api.Status status = response.getStatus();
+        if (status.getCode() != 0) {
+            throw new DKVException(status, "MultiPut", new Object[]{items});
+        }
+    }
+
+    @Override
+    public void put(KV.Bytes... items) {
+        Api.MultiPutRequest.Builder multiPutReqBuilder = Api.MultiPutRequest.newBuilder();
+        for (KV.Bytes kv : items){
+            Api.PutRequest.Builder putReqBuilder = Api.PutRequest.newBuilder();
+            Api.PutRequest putReq = putReqBuilder
+                    .setKey(copyFrom(kv.getKey()))
+                    .setValue(copyFrom(kv.getValue()))
+                    .setExpireTS(kv.getExpiryTS())
+                    .build();
+            multiPutReqBuilder.addPutRequest(putReq);
+        }
+        Api.MultiPutRequest multiPutReq = multiPutReqBuilder.build();
+        Api.PutResponse response = blockingStub.multiPut(multiPutReq);
+        Api.Status status = response.getStatus();
+        if (status.getCode() != 0) {
+            throw new DKVException(status, "MultiPut", new Object[]{items});
+        }
```

Comment:
same - can we avoid code duplication ?

Reply:
The duplicate code is just 2 lines which is the calling of grpc method. Extracting it to separate function will increase code footprint itself :)